### PR TITLE
Resizable table columns in Sessions

### LINKED ISSUE #3700: Resizable table columns
## Describe the solution you'd like
Resizable table columns so that you can view longer strings completely

## Additional context
image

### PR DESCRIPTION
**Pull request by samwinslow**
Fixes #3700.

## Changes

- Created new `ResizableTable` component that wraps ant-design's `Table` and provides column resize functionality.
- To use the ResizableTable, the caveat is that the sizes must be fixed to begin with, but this is handled intelligently by setting the `span` attribute on `columns` and responsive breakpoints.
- `span` is a number based on ant-design's [24-column system](https://ant.design/components/grid/).

Currently this new table is only used for Sessions, and it can be reused elsewhere

*If this affects the frontend, include screenshots.*  

https://user-images.githubusercontent.com/4645779/114090718-c9e17380-9885-11eb-8ec5-6544880ea5d7.mp4

## Checklist

- [x] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [x] Jest frontend tests
- [x] Cypress end-to-end tests
- [x] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious

### CODE REVIEW

**Comment by samwinslow** [comment]
Failing tests seem to be unrelated to this feature.

**Comment by Twixes** [comment]
Fixed a source of Cypress flakiness today #3892, so that should now be better for PRs in sync with master, though the situation is still not perfect on this front.
As for the Django tests run, yeah, GH Actions failure, Microsoft spirit is eating into GitHub and the latter has been getting flakier later… BTW you can always rerun checks by clicking on check "Details" and then "Re-run jobs", should make things green in most such situations.

File: frontend/src/lib/components/ResizableTable/responsiveUtils.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+function getPixelValue(cssStatement: string): number {
+    return parseInt(cssStatement.replace(/\D/g, ''), 10)
+}
+
+export function getActiveBreakpoint(): number {
+    const { innerWidth: width } = window
+    const breakpoints = Object.values(responsiveMap).map((cssStatement) => getPixelValue(cssStatement))
+    let breakpoint = breakpoints[0]
```
Comment (EDsCODE) [inline frontend/src/lib/components/ResizableTable/responsiveUtils.ts:28]: Is this a guaranteed non-empty array?

**Review by EDsCODE — COMMENTED** [review]
Awesome work! This is super useful. Two thoughts:

1. Could you set the default widths such that all the columns are initially visible at least in the case for the session table? It's nice to be able to immediately understand that you can play a recording but right now it's hidden off to the right. You need to scroll to it to be able to use it

2. Definitely followup with a PR after this one is merged and add this table to the event and persons tables

File: frontend/src/lib/components/ResizableTable/responsiveUtils.ts (hunk 1 above)
Comment (samwinslow) [reply frontend/src/lib/components/ResizableTable/responsiveUtils.ts:28]: @EDsCODE Yep, it's a direct import of an object from the `antd` library. Additionally, the ternary statements in `getMinColumnWidth` and `getMaxColumnWidth` essentially mean that if breakpoint were somehow undefined, a fallback value is guaranteed in those functions.

**Pull request merged**